test/verify/check-machines-dbus

```diff
@@ -594,6 +594,7 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.wait_in_text("#vm-subVmTest1-state", "paused")
         b.wait_attr("#delete-subVmTest1-disk-vdf", "disabled", "")
         m.execute("virsh resume subVmTest1")
+        wait(lambda: "login as 'cirros' user" in  m.execute("cat /var/log/libvirt/console-subVmTest1.log"), delay=3)
```

Comment:
What does this do?

Reply:
Waitd for the VM to get in the login prompt again. I figured that when resuming the VM and performing directly device attaching/detaching the OS takes longer time to respond.
And I think that made our test very flaky.

Reply:
Oh so until there is login prompt, attaching/detaching may get ignored? Thanks for fix